src/integrationTest/java/uk/gov/hmcts/reform/professionalapi/RetrieveOrganisationByProfileIdsIntegrationTest.java

```diff
@@ -0,0 +1,271 @@
+package uk.gov.hmcts.reform.professionalapi;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import uk.gov.hmcts.reform.professionalapi.controller.request.OrgAttributeRequest;
+import uk.gov.hmcts.reform.professionalapi.controller.request.OrganisationByProfileIdsRequest;
+import uk.gov.hmcts.reform.professionalapi.controller.request.OrganisationOtherOrgsCreationRequest;
+import uk.gov.hmcts.reform.professionalapi.controller.response.MultipleOrganisationsResponse;
+import uk.gov.hmcts.reform.professionalapi.controller.response.OrganisationByProfileResponse;
+import uk.gov.hmcts.reform.professionalapi.repository.OrganisationRepository;
+import uk.gov.hmcts.reform.professionalapi.util.AuthorizationEnabledIntegrationTest;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.hmcts.reform.professionalapi.controller.request.ContactInformationCreationRequest.aContactInformationCreationRequest;
+import static uk.gov.hmcts.reform.professionalapi.controller.request.DxAddressCreationRequest.dxAddressCreationRequest;
+import static uk.gov.hmcts.reform.professionalapi.controller.request.UserCreationRequest.aUserCreationRequest;
+
+class RetrieveOrganisationByProfileIdsIntegrationTest extends AuthorizationEnabledIntegrationTest {
+
+    private final String solicitorOrgType = "SOLICITOR-ORG";
+    private final String solicitorProfileId = "SOLICITOR_PROFILE";
+    private final String ogdHoOrgType = "OGD-HO-ORG";
+
+    @Autowired
+    private OrganisationRepository organisationRepository;
+
+    @BeforeEach
+    public void setup() {
+        organisationRepository.deleteAll();
```

Comment:
organisationRepository.deleteAll();
this will delete all organisations in test environment, it will cause issues

Reply:
From what I've understood, the database is an in-memory throwaway so it should not impact any hosted database in any environment